### ISSUE TITLE
START: 프로젝트 구조 및 config 구성

### ISSUE THREAD

**Issue by coouir** [LighTrip/LighTrip-AI#62]
# START: 프로젝트 구조 및 config 구성

## 목적

AI 모델 개발을 위한 기본 프로젝트 구조와 공통 설정 파일을 구성합니다.

## 작업 내용

- [ ] 기본 프로젝트 디렉터리 구조 생성
- [ ] `configs/default.yaml` 생성
- [ ] 입력 이미지 크기 설정
- [ ] ROI crop 좌표 설정
- [ ] 제목 위치 정보 설정
- [ ] 팔레트 색상 개수 설정
- [ ] contrast threshold 설정
- [ ] 학습 기본 hyperparameter 설정
- [ ] random seed 설정

## 권장 디렉터리 구조

    project/
      configs/
        default.yaml
      data/
        raw/
        processed/
          clean_images/
          rois/
          masks/
          labels/
        splits/
      src/
        data/
        labeling/
        models/
        training/
        evaluation/
        inference/
      notebooks/
      outputs/
        checkpoints/
        reports/
        previews/
      README.md

## 기본 config 예시

    input_size:
      height: 128
      width: 256

    roi:
      x1: 0.05
      y1: 0.15
      x2: 0.95
      y2: 0.50

    title:
      center_x: 0.5
      center_y: 0.333

    palette:
      num_colors: 96

    labeling:
      temperature: 0.2
      contrast_threshold: 4.5
      min_contrast_p05: 3.0

    training:
      batch_size: 64
      learning_rate: 0.0003
      weight_decay: 0.0001
      epochs: 20
      seed: 42

## 산출물

- `configs/default.yaml`
- 기본 프로젝트 디렉터리
- 초기 `README.md`

## 완료 기준

- [ ] config 파일이 생성되어 있다.
- [ ] 데이터, 학습, 평가, 추론 코드에서 공통 config를 사용할 수 있다.
- [ ] 주요 산출물 저장 경로가 정리되어 있다.

---
